CHANGES.md

```diff
@@ -3,6 +3,7 @@
 ## unreleased
 
 - removed jgit dependency, used com.lovelysystems.gradle.LSGit instead
+- add --merged flag to latestLocalGitTagVersion 
```

Comment:
pls try to describe the functional change, not what you actualy did in code, so that a user of the plugin undestands it.